pages/en/tokenomics.mdx

```diff
@@ -0,0 +1,102 @@
+
+## Creating a subgraph
+
+Developers can [create a subgraph](https://thegraph.com/docs/en/developing/creating-a-subgraph/) to index data on the blockchain. Subgraphs are instructions for Indexers about which data should be served to consumers.
+
+Once developers have built and tested their subgraph, they can [publish their subgraph](https://thegraph.com/docs/en/publishing/publishing-a-subgraph/) on The Graph's decentralized network.
+
+## Querying an existing subgraph
+
+Once a subgraph is [published](https://thegraph.com/docs/en/publishing/publishing-a-subgraph/) to The Graph's decentralized network, anyone can create an API key, fill it with GRT, and query it.
+
+Subgraphs are [queried using GraphQL](https://thegraph.com/docs/en/querying/querying-the-graph/), and the query fees are paid for with GRT in [Subgraph Studio](https://thegraph.com/studio/). Query fees are distributed to network participants based on their contributions to the protocol.
+
+1% of the query fees paid to the network are burned.
+
+# Indexers (Earn GRT)
+
+Indexers are the backbone of The Graph. They operate independent hardware and software to power the network. Indexers serve data to consumers based on instructions from subgraphs.
+
+Indexers can earn GRT rewards in two ways:
+
+1.  Query fees: GRT paid by developers or users for subgraph data queries. Query fees are deposited into a rebate pool and distributed to Indexers.
+
+2.  Indexing rewards: the 3% annual issuance is distributed to Indexers based on the number of subgraphs they are indexing. Indexing rewards incentivize Indexers to index subgraphs, sometimes before the query fees begin to accrue, and to submit Proofs of Indexing (POIs), which verify that they have indexed data accurately.
+
+Each subgraph is allotted a portion of the total network token issuance, based on the amount of curation signal that subgraph has. That amount is then divided between all the Indexers staked on that subgraph based on their allocated stake.
+
+In order to run an indexing node, Indexers must stake 100,000 GRT or more with the network. Indexers are incentivized to stake GRT in proportion to the amount of queries they serve.
+
+Indexers can increase their GRT allocations on subgraphs by accepting GRT delegation from Delegators, and they can accept up to 16X GRT of their initial. If an Indexer becomes "over-delegated" (i.e., more than 16X their initial stake), they will not be able to use the additional GRT from Delegators until they increase their stake in the network.
+
+The amount of rewards an Indexer receives can vary based on the initial stake, accepted delegation, quality of service, and many more factors. The following chart is publicly available data from an active Indexer on The Graph's decentralized network.
+
+## The Indexer stake & reward of allnodes-com.eth
+
+![Indexing stake and rewards](/img/indexing-stake-and-income.png) This data is from February 2021 to September 2022.
```

Comment:
```suggestion
![Indexing stake and rewards](/img/indexing-stake-and-income.png)

This data is from February 2021 to September 2022.
```

Doesn't change much (if anything) visually because images are forced to be full width at the moment, so the text wraps to the next line but it is technically inline with the image in your version. This puts it on the next paragraph, which I feel is more correct.